src/v2/Components/Artwork/__tests__/Details.jest.tsx

```diff
@@ -207,6 +210,109 @@ describe("Details", () => {
       const html = wrapper.html()
       expect(html).not.toContain("bid")
     })
+
+    describe("lot close info", () => {
+      it("shows the lot is closed if the lot end time has passed and if the sale has cascading end times enabled", async () => {
+        const data = {
+          ...artworkInAuction,
+          sale_artwork: {
+            // @ts-expect-error PLEASE_FIX_ME_STRICT_NULL_CHECK_MIGRATION
+            ...artworkInAuction.sale_artwork,
+            endAt: "2022-03-11T12:33:37.000Z",
+          },
+          sale: {
+            // @ts-expect-error PLEASE_FIX_ME_STRICT_NULL_CHECK_MIGRATION
```

Comment:
More generally tho, why adding all of these ignores (that's generally a code smell). That said tho, I'm (personally) totally impartial about this kind of thing in tests, and since we're using our old relay test tools here making sure everything is typed is basically just creating more tech-debt. 

A better fix would be to do something like:

```ts
const data = {
  foo: {
    bar: {
      baz: "hi"
    }
  }
} as any
```
or 
```ts
const data: any = {
  foo: {
    bar: {
      baz: "hi"
    }
  }
}
```

Reply:
Nice yeah that's better thanks!